TestingMap.md

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
love these notes/maps you make for yourself. You will really appreciate these when you come back to these projects later!!!